css/style.css

```
@@ -0,0 +1,415 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+  src:
+  url("../fonts/roboto.woff2") format("woff2"),
+  url("../fonts/roboto.woff") format("woff");
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family:"Roboto", sans-serif;
+  src:
+   url("../fonts/robotomedium.woff2")format("woff2"),
+   url("../fonts/robotomedium.woff") format("woff");
+   font-weight: 500;
+   font-style: normal;
+}
+@font-face {
+  font-family: "Roboto", sans-serif;
+  src:
+   url("../fonts/robotobold.woff2")format("woff2"),
+   url("../fonts/robotobold.woff") format("woff");
+   font-weight: 700;
+   font-style: normal;
+}
+
+
+
+
+:root {
+  --basic-green: #849d8f;
+  --basic-gray: #8996a6;
+  --basic-brown: #9d8b84;
+  --basic-orange: #e84d37;
+  --basic-white: #ffffff;
+  --basic-black: #323232;
+  --basic-gray-light: #999999;
+  --basic-gray-dark: #5a5a5a;
+  --basic-orange-light: #ffbc9e;
+  --basic-white-dark: #F8F7F4;
+  --special: #D07058;
+  --special-light: #FBDED7;
+  --special-dark: #F6B5A5;
+  --special-link: #EDEDED;
+  --pattern: #eadcbe;
+  --raspberry-background: #73172b;
+  --chocolate-bacground: #3a2217;
+}
```

Отступ между селекторам
```suggestion
}

```